apps/web/src/components/SearchAddressInput/index.tsx

```diff
@@ -31,12 +29,13 @@ export default function SearchAddressInput({ onChange }: SearchAddressInputProps
   /* 2. User enters an Basename */
   const validBasename = isBasename(value);
 
-  const { basenameChain } = useBasenameChain(value as BaseName);
+  const { chain: basenameChain } = useAccount();
+  const resolverAddress = basenameChain?.id ? USERNAME_L2_RESOLVER_ADDRESSES[basenameChain.id] : '0x';
```

Comment:
if the user is connected to L1 then this won't be a basename chain, right? It'd just be any chain, and the resolver address will be `0x` if not base/basesepolia?

if we're setting  universalResolverAddress to 0x and the chainId to undefined or mainnet, what's the behavior here?